assemblies/assembly-rhdh-integration-aws.adoc

```diff
@@ -10,6 +9,9 @@ The integration with {aws-short} requires the deployment of {product-short} in {
 * The {product} Operator
 
 include::modules/admin/proc-rhdh-deploy-eks-using-helm.adoc[leveloffset=+1]
+
+include::modules/installation/proc-installing-rhdh-on-kubernetes-with-helm-cli.adoc[leveloffset=+2]
```

Comment:
two questions:
* why is this in `installation/` instead of `admin/` and
* if we already have doc for helm CLI install for the two supported flavours of k8s (eks and aks) why do we ALSO need a generic install guide for some other flavour of k8s ... which we don't support?

Reply:
> why is this in installation/ instead of admin/
The directory in which the files are stored is arbitrary and based on the preference of the docs team, but installation does makes the most sense, as all of this content will be relocated from the admin guide to the installation guide in 1.2.1. 

>  why do we ALSO need a generic install guide for some other flavour of k8s ... which we don't support?
What flavor of k8s are you referring to? This content was already reviewed and merged under a different PR, so if it covers anything we don't support, that issue should have been addressed prior to merging. This PR simply changes the location of this content so that we don't have a k8s module under an OCP assembly. And it is a short-term fix since we will restructure and update content again when constructing the new installation guide. 

Some of these comments are helpful, but out of scope for this particular PR. Let's add some Jiras to [the Epic for improving the structure and content of the existing installation ](https://issues.redhat.com/browse/RHIDP-2701) as part of the development of the new _Installing RHDH_ title.

Reply:
Should we remove the k8s section entirely? It sounds like, if it's duplicating procs and commands, it isn't actually needed. But again, that should have been addressed during the review of [the PR that created this content](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/287).